ImportError C extension of pandas

After a fresh install of this package on a Ubuntu 18.04 machine, 12 tests fail due to a missing C extension of pandas.

This did not seem to be an issue in [the last build](https://travis-ci.org/github/peldszus/evidencegraph/builds/567643048).

The details of the error are:
```
E   ImportError: C extension: /tmp/eg/evidencegraph/env/local/lib/python2.7/site-packages/pandas/hashtable.so: undefined symbol: pyobject_cmp not built. If you want to import pandas from the source directory, you may need to run 'python setup.py build_ext --inplace' to build the C extensions first.
```

After updating `pandas` to the next minor release, i.e. to `0.18.1`, this problem seems to be resolved.

--- a/.pre-commit-config.yaml
+++ b/.pre-commit-config.yaml
@@ -1,6 +1,6 @@
 repos:
   - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: master
+    rev: v2.5.0
     hooks:
       - id: check-json
       - id: check-merge-conflict

--- a/requirements.txt
+++ b/requirements.txt
@@ -3,7 +3,7 @@ pytest-cov==2.6.0
 numpy==1.10.2
 scikit-learn==0.15.2
 scipy==0.14.0
-pandas==0.17.1
+pandas==0.18.1
 networkx==1.9
 bidict==0.1.5
 matplotlib==1.5.1

